src/redux/session.js

```diff
@@ -103,3 +214,52 @@ module.exports.refreshSession = () => (dispatch => {
         return;
     });
 });
+
+module.exports.handleLogIn = (formData, callback) => (dispatch => {
+    dispatch(module.exports.setLoginError(null));
+    formData.useMessages = true; // NOTE: this may or may not be being used anywhere else
+    api({
+        method: 'post',
+        host: '',
+        uri: '/accounts/login/',
+        json: formData,
+        useCsrf: true
+    }, (err, body) => {
+        if (err) dispatch(module.exports.setLoginError(err.message));
+        if (body) {
+            body = body[0];
+            if (body.success) {
+                dispatch(module.exports.setLoginOpen(false));
+                body.messages.map(message => { // eslint-disable-line array-callback-return
```

Comment:
Maybe this is copy-pasted, but I think we should heed this linter warning instead of silencing it. We shouldn't use `map` when we mean to do a `for`.

Reply:
You're right that this was copy-pasted, but it's still good to fix the stuff I touch!

You're also right that map is wrong here. Changed it to forEach and removed the linter override.